lib/utils/escape-filter-value.js

```diff
@@ -27,22 +27,22 @@ function escapeFilterValue (toEscape) {
 }
 
 function escapeBuffer (buf) {
-  let result = ''
+  let bufLen = buf.length;
```

Comment:
Please use `.byteLength` to be sure we account for multi-byte characters:
```suggestion
  let bufLen = buf.byteLength;
```

Reply:
OK, I will change it.